factors.qmd

```diff
@@ -421,9 +421,9 @@ In practice, `ordered()` factors behave very similarly to regular factors.
 There are only two places where you might notice different behavior:
 
 -   If you map an ordered factor to color or fill in ggplot2, it will default to `scale_color_viridis()`/`scale_fill_viridis()`, a color scale that implies a ranking.
--   If you use an ordered function in a linear model, it will use "polygonal contrasts". These are mildly useful, but you are unlikely to have heard of them unless you have a PhD in Statistics, and even then you probably don't routinely interpret them. If you want to learn more, we recommend `vignette("contrasts", package = "faux")` by Lisa DeBruine.
+-   If you use an ordered predictor in a linear model, it will use "polynomial contrasts". These are mildly useful, but you are unlikely to have heard of them unless you have a PhD in Statistics, and even then you probably don't routinely interpret them. If you want to learn more, we recommend `vignette("contrasts", package = "faux")` by Lisa DeBruine.
 
-Given the arguable utility of these differences, we don't generally recommend using ordered factors.
+Ordered categories are frequently used in the social sciences but not so common in other fields, so unless you are faced with a dataset that has clearly ranked categories we don't generally recommend using ordered factors.
```

Comment:
What's the impact on analysis of using them? Is it that the social sciences are using packages that have more behaviour differences between factors and ordered factors?

Reply:
Yeah, my understanding is that as social sciences have so many survey-based data in a ranked scale, they use more often ordinal regression and other similar methods to deal with ordered categories both as predictors and as responses. For response variables, there is quite a bit of literature on ordinal regressions. these usually employ cumulative logit distributions. Methods to deal with ordered factors as predictors in a better way than these polynomial contrasts do seem to be much more recent, but are getting increasing use, at least in Bayesian analyses (se https://bpspsychub.onlinelibrary.wiley.com/doi/10.1111/bmsp.12195 for example).